build.gradle

```diff
@@ -26,11 +35,29 @@ repositories {
 }
 
 dependencies {
-  implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
```

Comment:
data-jpa non serve?

Reply:
forse no. In ogni caso entra in conflitto con webflux, quindi o si disabilitano dei bean dentro a jpa o non compila niente.

me ne occuperò su una branch a parte così non vi blocco il lavoro. 

fate attenzione ad aggiungere il MINIMO di dipendenze indispensabili! consiglierei di aggiungere le dipendenze UNA ALLA VOLTA perché io ci ho messo giorni a capire quale dava problemi